sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -187,27 +189,47 @@ protected override async Task<bool> AcceptEvent (SessionId sessionId, string met
 					break;
 				}
 
+			case "Debugger.breakpointResolved": {
+					break;
+				}
+
 			case "Debugger.scriptParsed":{
 					var url = args? ["url"]?.Value<string> () ?? "";
 
 					switch (url) {
 					case var _ when url == "":
 					case var _ when url.StartsWith ("wasm://", StringComparison.Ordinal): {
-							Log ("info", $"ignoring wasm: Debugger.scriptParsed {url}");
+							Log ("verbose", $"ignoring wasm: Debugger.scriptParsed {url}");
 							return true;
 						}
 					}
-					Log ("info", $"proxying Debugger.scriptParsed ({sessionId.sessionId}) {url} {args}");
+					Log ("verbose", $"proxying Debugger.scriptParsed ({sessionId.sessionId}) {url} {args}");
 					break;
 				}
 			}
 			return false;
 		}
 
+		async Task<bool> IsRuntimeReadyAlready (SessionId sessionId, CancellationToken token)
```

Comment:
Might be just me - I find the name confusing, I would suggest `CheckIsRuntimeReady` or something like that.

Reply:
I uh, changed it